packages/static-wado-webserver/lib/dicomWebServerConfig.mjs

```diff
@@ -1,23 +1,19 @@
 import ConfigPoint from "config-point";
 import StaticCreator from "@ohif/static-wado-creator";
+import "@ohif/static-wado-plugins";
 
 const { staticWadoConfig } = StaticCreator;
 
 /**
  * Defines the basic configuration values for the dicomwebserver component.  See the README for more details.
  */
 const { dicomWebServerConfig } = ConfigPoint.register({
-  plugins: {
-    // TODO - figure out where these should actually go
-    studyQueryReadIndex: "../lib/studyQueryReadIndex.mjs",
-  },
-
   dicomWebServerConfig: {
     configBase: staticWadoConfig,
     helpShort: "dicomwebserver",
     helpDescription:
       "Serve up the static wado files and optionally a web client as a web server on the local machine.",
-    studyQuery: "studyQueryReadIndex",
+    studyQuery: "studiesQueryByIndex",
```

Comment:
maybe the design should be:

dicomWebServerConfig -> plugins -> [{studyQuery: studiesQueryByIndex}]

Reply:
I'm trying to decouple the studyQuery from the load location and from the choice of plugin.  That allows on to define the load location in one spot, and choose how it gets used in other spots.  In specific, for Phoenix, that will allow once configuration file to define
SCP studyQuery being "studiesQueryByIndex"
Web studyQuery being "studiesQueryToPhoenixDB"
and then define each plugin location once.
That is, I'm encapsulating the volatility of the location of the plugin inside the "plugins" config point while the volatility of the studyQuery is encapsulated in the configuration for each element.

Reply:
What I meant was actually to do the following change:

dicomWebServerConfig = {
 plugins: [{studyQuery: 'studiesQueryByIndex'}]
 }
 I.e just group plugins on a property instead of having many as direct child of object

Reply:
For that one, I was keeping the two items as distinct settings because they do different things, and there are times one needs to choose one path for one type of item, and a different path for the other.  Thus, the "plugins" declaration declares where the plugins are locations, while the values inside the dicomWebServerConfig declare which plugin to use.
There is in fact a location where one can set the studyQuery more globally for different uses, staticWadoConfig,  but there may well be times when the same plugin gets used in different ways, or different things use the same name plugin (studyQuery), with with a different plugin applied.
I'm going to keep this one as is.